Undefined variable in util.go

Hopefully it's not because I'm not familiar with how go works, but when running:

../../../go/pkg/mod/github.com/muka/peerjs-go@v0.0.0-20210427075427-c4a7fe257084/util/util.go:45:41: undefined: ChunkedMTU
../../../go/pkg/mod/github.com/muka/peerjs-go@v0.0.0-20210427075427-c4a7fe257084/util/util.go:50:49: undefined: ChunkedMTU

Is there some step I'm missing? Thank you.

--- a/README.md
+++ b/README.md
@@ -1,4 +1,4 @@
-# Golang PeerJS 
+# Golang PeerJS
 
 A Golang port of [PeerJS](https://github.com/peers/peerjs)
 
@@ -16,17 +16,27 @@ See [_examples folder](./_examples)
 
 ```golang
 
-	peer1, _ := NewPeer("peer1", NewOptions())
+package main
+
+import (
+	"log"
+	"time"
+
+	peer "github.com/muka/peerjs-go"
+)
+
+func main() {
+	peer1, _ := peer.NewPeer("peer1", peer.NewOptions())
 	defer peer1.Close()
 
-	peer2, _ := NewPeer("peer2", NewOptions())
+	peer2, _ := peer.NewPeer("peer2", peer.NewOptions())
 	defer peer2.Close()
 
 	peer2.On("connection", func(data interface{}) {
-		conn2 := data.(*DataConnection)
+		conn2 := data.(*peer.DataConnection)
 		conn2.On("data", func(data interface{}) {
 			// Will print 'hi!'
-			log.Printf("Received: %v\n", data)
+			log.Printf("Received: %#v: %s\n", data, data)
 		})
 	})
 
@@ -38,7 +48,8 @@ See [_examples folder](./_examples)
 		}
 	})
 
-	select{}
+	select {}
+}
 ```
 
 ### Peer server

--- a/conn_data.go
+++ b/conn_data.go
@@ -15,8 +15,6 @@ const (
 	DataChannelIDPrefix = "dc_"
 	//MaxBufferedAmount max amount to buffer
 	MaxBufferedAmount = 8 * 1024 * 1024
-	// ChunkedMTU payload size for a single message
-	ChunkedMTU = 16300
 )
 
 // NewDataConnection create new DataConnection

--- a/util/util.go
+++ b/util/util.go
@@ -9,7 +9,11 @@ import (
 // Seed personal random source - faster and won't mess with global one
 var tokenRand = rand.New(rand.NewSource(time.Now().UnixNano()))
 
-const tokenChars = "1234567890abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+const (
+	tokenChars = "1234567890abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+	// ChunkedMTU payload size for a single message
+	ChunkedMTU = 16300
+)
 
 func RandomToken() string {
 	b := make([]byte, 11) // PeerJS random tokens are 11 chars long

